overhaul(everything): complete overhaul and update to docs and v6 (#26)
cross-seed/cross-seed.org: docs/basics/faq-troubleshooting.md

#### v6

If possible, `cross-seed` will always attempt to create links for data and torrents being injected into a client, unlike in v5 where torrent matches were directly linked to the data.

:::tip
If you wish to search only data, please read the relevant version's instructions.

-   [v5](#v5)
-   [v6](../v6-migration.md#updated-torrentdir-option)

:::

### Error: ENOENT: no such file or directory

This is usually caused by broken symlinks in, or files being moved or deleted from, your [`dataDirs`](./options.md#datadirs) during a data-based search.
Check the `/logs/verbose.*.log` files for the file causing this.

:::caution
If you do not link files within your `dataDirs` or have them outside of the [`maxDataDepth`](../tutorials/data-based-matching.md#setup) visibility, this is preventable.
:::

### My torrents are injected (qBittorrent) but show `missing files` error!

> If you are seeing injected torrents show up with `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning when the torrents are being injected telling you that there is no save path set.

Author: the difference between tip and caution is only the icon for the box, and the color. You can add whatever word you want in place of Tip by just doing the following..

`:::tip Caution`

is there a reason you want this color changed? It's a tip imo

> If you are seeing injected torrents show up with `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning when the torrents are being injected telling you that there is no save path set.

Reviewer: I'm not asking for a color change. I'm asking about the paragraph of text where is says:
> You should also see a warning when the torrents are being injected telling you that there is no save path set

> If you are seeing injected torrents show up with `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning when the torrents are being injected telling you that there is no save path set.

Author: Oh sorry, I just woke up...I thought that was the suggestion lol

It'd be in the logs that the warning would exist for cross-seed, or where the progress bar is in qbittorrent...

> If you are seeing injected torrents show up with `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning when the torrents are being injected telling you that there is no save path set.

Reviewer: ```suggestion
If you see injected torrents show up with a `missing files` error, it is likely because you do not have a save path set for the original torrent's category that `cross-seed` was cross-seeding from. You should also see a warning in the logs when the torrents are injected telling you there is no save path set.
```

Using [`safe`](./options.md#matchmode) matching will match torrents for snatching based on the size and require that the release groups match and compare them, requiring file structure to match.
:::

:::danger WARNING

> There are TV shows that are not combined into season packs, such as Jeopardy, Talk Shows, or similar. These shows often have naming which specifies dates, rather than SeasonEpisode style. Searching with `includeSingleEpisodes` and using `risky` on these torrents, or files, can result in _many torrent files being downloaded_ for comparisons for matching only _one individual episode_.

Reviewer: ```suggestion
There are TV shows that are not combined into season packs, such as Jeopardy, Talk Shows, and similar. These shows are known as dailies and often have naming which specifies dates, rather than SeasonEpisode (S02E03) style. Searching with `includeSingleEpisodes` and using `risky` on these torrents, or files, can result in _many torrent files being downloaded_ for comparisons for matching only _one individual episode_.
```

Remember, this would occur on just one (torrent) search, within the entire (all data) search.
:::

---

##### Final Notes

> :::tip Be ADVISED

Reviewer: We should probably stick with all caps (BE ADVISED) or all headline case (Be Advised)

Author: It only display as uppercase, you could put BE AdViSeD and it will still show all caps in the render.

> **There is no one-size-fits-all setting for this**, but you can run `cross-seed` with your config and see how many torrents it will search (noted as "suitable" prior to conducting the actual search) and adjust accordingly.

Reviewer: ```suggestion
**There is no one-size-fits-all setting for this**. You can run `cross-seed` with your config to see how many torrents it will search (noted as "suitable" before conducting the actual search) and adjust accordingly.
```
Remove "but". People ignore everything before the "but"

:::

-   It is highly recommended to use [`excludeOlder` and `excludeRecentSearch`](./daemon.md#set-up-periodic-searches). Both of these are covered [here](./daemon.md#set-up-periodic-searches) as well as on the [options page](./options.md#excludeolder).

>     -   Setting something reasonable for both of these based on the amount of content you download is crucial. While we cache your .torrents, it is not beneficial, for example, to repeatedly search the same torrents that are 6 months old every few days.

Reviewer: Spacing seems off here compared to the point above and below it.

Reviewer: If this is supposed to be a sublist item ignore the comment above.

Author: it's supposed to be this way.
